ext/intl/calendar/calendar_methods.cpp

```diff
@@ -355,6 +362,7 @@ static void _php_intlcal_before_after(
 	CALENDAR_METHOD_FETCH_OBJECT;
 
 	when_co = Z_INTL_CALENDAR_P(when_object);
+	/* Can this ever happen ? */
```

Comment:
Should be possible, e.g. via newInstanceWithoutConstructor.

Reply:
Ah yeah, does are usually Errors right?

Reply:
Yeah, we use Error for uninitialized objects.